vscode-extension/package.json

```diff
@@ -56,13 +56,13 @@
         "category": "AIConfig"
       },
       {
-        "command": "vscode-aiconfig.createAIConfigJSON",
-        "title": "Create New AIConfig (json)",
+        "command": "vscode-aiconfig.createExampleAIConfigJSON",
+        "title": "Create New Example AIConfig (json)",
         "category": "AIConfig"
       },
       {
-        "command": "vscode-aiconfig.createAIConfigYAML",
-        "title": "Create New AIConfig (yaml)",
+        "command": "vscode-aiconfig.createExampleAIConfigYAML",
+        "title": "Create New Example AIConfig (yaml)",
```

Comment:
The main user-facing change is here.

cc @tanya-rai , @saqadri what do you think/prefer? 

Next PR Ankush is implementing a "real" empty new AIConfig

Reply:
nit, having "Create New E..." as text for both could lead to accidentally picking one when wanting the other. Maybe this could be 'Create AIConfig from Example" and next command can be "Create New AIConfig"?

Reply:
I like this. Updated